feature-tests/README.md

```diff
@@ -20,15 +20,12 @@ Set up the following env vars.
 ```shell
 cd feature-tests
 npm install && npm build
-export TEST_ENVIRONMENT=dev
-export SAM_STACK_NAME=stackName
-export tagFilter=@regression
 ```
 
 To run all API tests locally in Jest via Jest.
 
 ```shell
-npm test
+tagFilter=@regression npm test
```

Comment:
should this not be `tagFilter=@pre-merge npm test`

Reply:
Thanks Seamus, Readme file has been updated to show the correct test tag for running pre-merge tests